samples/DirectX/Program.cs

```diff
@@ -14,7 +14,7 @@ public static unsafe class Program
             new HelloWindow11(1280, 720, "D3D11.HelloWindow"),
             new HelloTriangle11(1280, 720, "D3D11.HelloTriangle"),
             new HelloWindow12(1280, 720, "D3D12.HelloWindow"),
-            new HelloTriangle12(1280, 720, "D3D12.HelloTriangle"),
+            new HelloTriangle12(720, 720, "D3D12.HelloTriangle"),
```

Comment:
Intended for checkin?

Reply:
I thought I'd unstaged them. They were for debugging something else